lib/internal/webstreams/util.js

```diff
@@ -139,23 +140,10 @@ function transferArrayBuffer(buffer) {
   return res;
 }
 
-function isDetachedBuffer(buffer) {
-  if (ArrayBufferGetByteLength(buffer) === 0) {
```

Comment:
In most practical use cases, pre-checking `ArrayBufferGetByteLength(buffer) !== 0` is enough to find out the detachment. Since a call into C++ from JS is generally slow, leaving this part in JS would get better performance.

Reply:
To be fully correct, though, there would still need to be another check when the byteLength was zero, right? (which is surely more straightforward in C++, but could potentially be done in JS as well)

Reply:
Right. I meant that we can judge a `false` case without a C++ call or another check if the byteLength isn't zero in JS.

Reply:
Thanks for your reviews. I'm updating the pull request.